static/js/graphs_page.js

```diff
@@ -2736,6 +2862,7 @@ var cytoscapeGraph = {
          /*
          Function to Show information about the graph that the layout editor hid from the users.
          */
+        console.log(cy.style().selector('node'));
```

Comment:
Can you remove the log.

Reply:
Yeah sure! Actually I initially was making changes on master branch and they were already there.